# The cmake configuration file provided my upstream is not installed in the
# appropriate location. This patch moves it to the proper place.

--- a/src/core/CMakeLists.txt
+++ b/src/core/CMakeLists.txt
@@ -124,7 +124,8 @@ if (SOCI_STATIC)
     ARCHIVE DESTINATION ${LIBDIR})
 endif()
 
-install(EXPORT SOCI NAMESPACE SOCI:: DESTINATION cmake)
+install(EXPORT SOCI NAMESPACE SOCI:: DESTINATION
+    ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/cmake)
 
 #
 # Core configuration summary
--- a/cmake/SociBackend.cmake
+++ b/cmake/SociBackend.cmake
@@ -229,7 +229,8 @@ macro(soci_backend NAME)
          )
       endif()
 
-      install(EXPORT SOCI NAMESPACE SOCI:: DESTINATION cmake)
+      install(EXPORT SOCI NAMESPACE SOCI:: DESTINATION 
+              "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/cmake")
 
     else()
         colormsg(HIRED "${NAME}" RED "backend disabled, since")
